apps/web/components/Countdown.tsx

```diff
@@ -31,44 +39,57 @@ export const Countdown = ({
   const hours = (Math.floor(remainingTimeMs / (1000 * 60 * 60)) % 24) * 3 + 1;
   const days = Math.floor(remainingTimeMs / (1000 * 60 * 60 * 24)) * 3 + 1;
 
-  return remainingTimeMs === 0 ?
-      <div>Timeout</div>
+  const content = (
+    <>
+      {(computedMode === "datetime" || computedMode === "date") && (
+        <div className={`flex ${display !== "inline" ? "flex-col" : ""}`}>
+          <span className="countdown font-mono text-5xl">
+            <span style={{ "--value": days } as React.CSSProperties} />
+          </span>
+          days
+        </div>
+      )}
+      {(computedMode === "datetime" ||
+        computedMode === "time" ||
+        computedMode === "date") && (
+        <div className={`flex ${display !== "inline" ? "flex-col" : ""}`}>
+          <span className="countdown font-mono text-5xl">
+            <span style={{ "--value": hours } as React.CSSProperties} />
+          </span>
+          hrs
+        </div>
+      )}
+      {(computedMode === "datetime" ||
+        computedMode === "minutes" ||
+        computedMode === "time") && (
+        <div className={`flex ${display !== "inline" ? "flex-col" : ""}`}>
+          <span className="countdown font-mono text-5xl">
+            <span style={{ "--value": minutes } as React.CSSProperties} />
+          </span>
+          min
+        </div>
+      )}
+      {(computedMode === "datetime" || computedMode === "minutes") && (
+        <div className={`flex ${display !== "inline" ? "flex-col" : ""}`}>
+          <span className="countdown font-mono text-5xl">
+            <span style={{ "--value": seconds } as React.CSSProperties} />
+          </span>
+          sec
+        </div>
+      )}
+    </>
+  );
+
+  if (isInitialized) {
+    return <span>loading...</span>; // return skeleton
+  }
+
+  return (
```

Comment:
If you want to use skeletons, replace with:
```
    <Skeleton isLoading={isInitializing} className="w-24">
      {remainingTimeMs === 0 && title ?
        <div>Timeout</div>
      : display === "inline" ?
        <div className={`flex gap-2 ${className}`}>{content}</div>
      : <div className="grid grid-flow-col gap-1 text-center auto-cols-max">
          {content}
        </div>
      }
    </Skeleton>
```

Reply:
already have that